src/braille/qml/MuseScore/Braille/BrailleView.qml

```diff
@@ -134,7 +138,7 @@ StyledFlickable {
         cursorDelegate: Rectangle {
```

Comment:
I'm not sure why we're using a [`cursorDelegate`](https://doc.qt.io/qt-5/qml-qtquick-textedit.html#cursorDelegate-prop) here. I tried deleting this whole section:
```QML
        cursorDelegate: Rectangle {
            ...
        }
```
With that gone, the built-in cursor worked just fine on Windows, and was the correct colour in light and dark modes.

Please delete the `cursorDelegate` section and push the change as a new commit. Once the CI builds are finished, we'll test on each OS to make sure the built-in cursor works, then ask you to squash the commits.

Reply:
Actually when we press 'N'(for Braille Input Mode) the cursor color is changed to green, so for that we need it. If we remove the `cursorDelegate` section it will remain black or white only when we press 'N'

Reply:
True! Well spotted!

Reply:
> Please delete the `cursorDelegate` section and push the change as a new commit. Once the CI builds are finished, we'll test on each OS to make sure the built-in cursor works, then ask you to squash the commits.

Note that as far as I can tell, the braille input does not work on Linux, or at least didn't work prior to this PR.  See https://github.com/musescore/MuseScore/issues/20221.  I'd love to find out it's just my systems where it fails and that it does work in general.

Reply:
@MarcSabatella The braille input seems to work on my machine prior to this PR.

Reply:
OK, thanks for the info!  To be clear - you are saying it doesn't just display, but actually allows input?  What distribution is that, and which window manager?  And is it working for you in 4.2.1, or only in builds from master?

Reply:
@MarcSabatella Yes, I can input the quarter note C using FJKL. I am currently using Manjaro Linux with XFCE. Yup, its working on 4.2.1 and also on master.

https://github.com/musescore/MuseScore/assets/105774228/cf0a5207-f1c3-4599-9c4e-6a55a497c07a